features/crashprobe.feature

```diff
@@ -113,7 +113,9 @@ Feature: Reporting crash events
     And I configure Bugsnag for "SwiftAssertion"
     And I wait to receive a request
     Then the request is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
-    And the exception "errorClass" equals "Fatal error"
+    And the exception "errorClass" equals one of:
```

Comment:
It'd be good to include a comment referencing the ticket number which will ultimately provide a more long-term fix for this